Shared/Realm.Shared/Linq/IRealmCollection.cs

```diff
@@ -113,6 +113,7 @@ public interface IRealmCollection<out T> : IReadOnlyList<T>, INotifyCollectionCh
         /// Gets the <see cref="ObjectSchema"/> of the contained objects.
         /// </summary>
         /// <value>The ObjectSchema of the contained objects.</value>
+        [Obsolete("This property is no longer necessary and will be removed in a subsequent release.")]
```

Comment:
It's useful in meta-programming - either when working with the dynamic API, or when you're handed a downcasted type, such as `IRealmCollection<RealmObject>` (or some other type once we support inheritance).